examples/the-main-function/the_main_function.out

```diff
@@ -1,18 +1,6 @@
 # To run this sample, navigate to the directory that contains the
 # `.bal` file, and execute the `bal run` command below.
 
-# Use the ballerina `run` command to invoke the `main` function specifying `Alice`
-# as the string argument for `name`. `18` would be set as the value for
-# `age` and `Freshman` would be set as the value for `year`.
-bal run the_main_function.bal -- Alice
-Name: Alice, Age: 18, Year: Freshman
-
-# Use the ballerina `run` command to invoke the `main` function specifying `Alice`
-# as the string argument for `name` and `20` as the integer value for
-# `age`. Both arguments are specified as operands.
-bal run the_main_function.bal -- Alice 20
-Name: Alice, Age: 20, Year: Freshman
-
```

Comment:
bal run the_main_function.bal -- Ali
error: InvalidName {"message":"invalid length"}

is this correct?

Reply:
Fixed